.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: rust
 sudo: required
+dist: trusty
```

Comment:
Nah apparently this was needed to get CI to pass, I guess Travis updated some GCE images to 12.04 instead of 14.04?  Not entirely sure why some of the builders went back in time.